### PR TITLE
Ciid Hero

### PR DESCRIPTION
**Pull request by Zordas**
Cidd hero for Epic Seven

### CODE REVIEW

**Comment by Zordas** [comment]
Epic Seven Hero addition

File: hero/cidd.json [hunk 1]
```diff
@@ -0,0 +1,199 @@
+{
+    "name": "Cidd",
+    "rarity": 5,
+    "classType": "Thief",
+    "element": "Earth",
+    "zodiac": "Aries",
```
Comment (RaphaelDDL) [inline hero/cidd.json]: `classType`, `element` and `zodiac` values should be lowercase.

File: hero/cidd.json [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    "specialtyChangeName": "",
+    "selfSkillBarName": "",
+    "background": [
+        "Cidd is the youngest member and mascot of the Wild Dog Company, a group of mercenaries headquartered in an old ruined fortress in Wetheric Moor. Contrary to his cheerful and carefree appearance, he will complete any task assigned to him. He cares for the Wild Dog Company as if they were family."
+    ],
+    "relations": [
+        { "hero": "Dingo", "relationType": "trust" },
+        { "hero": "Otillie", "relationType": "trust" },
+        { "hero": "Armin", "relationType": "Rival" }
```
Comment (RaphaelDDL) [inline hero/cidd.json]: `relations`' hero names and types values should be lowercase.

File: hero/cidd.json [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+            "isPassive": false,
+			"soulBurn": 10,
+            "soulBurnEffect": "Increases damage dealt",
+            "awakenUpgrade": true,
+            "cooldown": 4,
+            "name": "Hack",
+            "soulAcquire": 2,
+            "description": "Cuts with a dagger, granting an extra turn if they enemy is defeated. Attacks with a more powerful skill when the caster is granted increased Speed.",
+            "enhancement": [
+                "Damage increased by 5%",
```
Comment (RaphaelDDL) [inline hero/cidd.json]: For enhancements, please follow game's text.

That is, `+5% damage dealt` or `-1 turn cooldown`

File: hero/cidd.json [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+        {
+            "rank": 1,
+            "skillUpgrade": false,
+            "statsIncrease": [{ "atk": "3%" }, { "atk": 20 }, { "hp": 60 }],
+            "resources": [{ "item": "life-rune", "qty": 8 }]
+        },
+        {
+            "rank": 2,
+            "skillUpgrade": false,
+            "statsIncrease": [{ "spd": "2" }, { "atk": 20 }, { "hp": 60 }],
```
Comment (RaphaelDDL) [inline hero/cidd.json]: spd increase is an integer, so it can be without quotes.

File: hero/cidd.json [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+        {
+            "rank": 5,
+            "skillUpgrade": false,
+            "statsIncrease": [{ "atk": "6%" }, { "atk": 30 }, { "hp": 80 }],
+            "resources": [{ "item": "epic-life-rune", "qty": 5 }, { "item": "blessing-of-orbis", "qty": 12 }]
+        },
+        {
+            "rank": 6,
+            "skillUpgrade": false,
+            "statsIncrease": [{ "spd": "4" }, { "atk": 30 }, { "hp": 80 }],
```
Comment (RaphaelDDL) [inline hero/cidd.json]: same as above for speed.

File: hero/cidd.json [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+                "Damage dealt by Relentless Strike increased by 5%",
+                "Damage dealt by Relentless Strike increased by 10%",
+                "Damage dealt by Relentless Strike increased by 15%"
+            ],
+            "buffs": ["stic_spd_up"],
+            "debuffs": []
+        },
+        {
+            "isPassive": false,
+			"soulBurn": 10,
```
Comment (RaphaelDDL) [inline hero/cidd.json]: Please fix the format to be on the same indentation as others.

**Pull request merged**